### DISCUSSION TITLE
Install script for Ubuntu

### DISCUSSION THREAD

**Discussion by yellow1912** [libvips/libvips#2644] [Show and tell]
I created this script, hope it will save other users some time:

https://gist.github.com/yellow1912/e528913433e83188bdc2608629f4618b

Thank you for creating libvips.

**Comment by jcupitt** [comment]
Oh nice, thanks for this.

There are some sample dockerfiles here too, for example:

https://github.com/jcupitt/docker-builds/blob/master/libvips-ubuntu21.04/Dockerfile
